code/io/joy-sdl.cpp

```diff
@@ -364,28 +393,24 @@ namespace joystick
 		return _id;
 	}
 
-	void Joystick::fillValues()
-	{
+	void Joystick::fillValues() {
```

Comment:
There are a bunch of format changes here that result in no functional difference.